### DISCUSSION TITLE
How to fix WiFi?

### DISCUSSION THREAD

**Discussion by atteneder** [medlor/bleeding-edge-ranciliopid#69] [Q&A] [answered]
Hi,

thanks for the work on the project. I've been running it smoothly for quite some time now. About a month ago I updated from 3.0.0 to the recent version (incl. updating the platform version) and roughly since then it does not connect to WiFi anymore.

How can I debug the WiFi connection wrt why it fails to connect?

I've double checked my `userConfig.h` and it should be fine.

This is unpleasant, since the cleaning mode requires WiFi and Blynq.

Thanks

**Comment by medlor** [comment]
Hi,
you are right. I have similar issues and thought there is an issue with my hardware.

I ll check this.

"This is unpleasant, since the cleaning mode requires WiFi and Blynq.": Really. Check this too.

**Comment by medlor** [comment, marked as answer]
I have added some wifi fixes in https://github.com/medlor/bleeding-edge-ranciliopid/tree/3.2.3_beta .
Feel free to check this out. This one is working much better for me.

**Reply by atteneder** [reply]
Indeed it resolved my problem. Thanks!

As someone with no experience with PlatformIO development I'd still be curious how to debug this project. Hints/docs would be greatly appreciated.

**Reply by medlor** [reply]
sorry but i am not using any sophisticated debugging stuff :-)
Just simple DEBUG_LOGS (see userConfig.h)
